Makefile.toml

```diff
@@ -926,6 +938,50 @@ pubsys \
 '''
 ]
 
+[tasks._upload-ova-base]
+# Rather than depend on "build", which currently rebuilds images each run, we
+# depend on publish-tools and check for the image files below to save time.
+# This does mean that `cargo make` must be run before `cargo make ami`.
+dependencies = ["setup-build", "publish-tools"]
+script_runner = "bash"
+script = [
+'''
+set -e
+
+export PATH="${BUILDSYS_TOOLS_DIR}/bin:${PATH}"
+
+ova_path="${BUILDSYS_OUTPUT_DIR}/${BUILDSYS_NAME_FULL}.ova"
+if [ ! -s "${ova_path}" ]; then
+   echo "An OVA doesn't exist for the current version/commit - ${BUILDSYS_VERSION_FULL} - please run 'cargo make'" >&2
+   exit 1
+fi
+
+vm_name="${VMWARE_VM_NAME:-${VMWARE_VM_NAME_DEFAULT}}"
+
+pubsys \
+   --infra-config-path "${PUBLISH_INFRA_CONFIG_PATH}" \
+   \
+   upload-ova \
+   \
+   --ova "${ova_path}" \
+   --spec "${VMWARE_IMPORT_SPEC_PATH}" \
+   --name "${vm_name}" \
+   \
+   ${MARK_OVA_AS_TEMPLATE:+--mark-as-template} \
+   \
+   ${VMWARE_DATACENTERS:+--datacenters "${VMWARE_DATACENTERS}"}
+'''
+]
+
+[tasks.upload-ova]
+script_runner = "bash"
+extend = "_upload-ova-base"
+
+[tasks.vmware-template]
+script_runner = "bash"
+env = { "MARK_OVA_AS_TEMPLATE" = "true" }
```

Comment:
I think this option should be explained with a comment above both upload-ova and vmware-template. It took me a minute to understand what was going on.

Reply:
Will do